doc/simultaneous_trajectory_execution/simultaneous_trajectory_execution_tutorial.rst

```diff
@@ -0,0 +1,47 @@
+Simultaneous Trajectory Execution
+==================================
+
+Introduction
+------------
+MoveIt now allows simultaneous execution of trajectories, as long as, each trajectory uses a different set of controllers. For example, in a dual arm environment, each arm can execute a different set of trajectories without needing to wait for the other arm to finish moving or manually synchronizing the motion of both arm into a single trajectory. Optionally, a collision check is performed right before execution of new trajectories to prevent collisions with active trajectories.
+
+The following GIF shows simple example of simultaneous execution of trajectories through the **Rviz Motion Planning** plugin.
+
+.. only:: html
```

Comment:
why is that needed?

Reply:
It is a very quick way to try the simultaneous feature from Rviz.

Reply:
I was refering to the `.. only:: html` only

Reply:
Oh, that. I guess it is not necessary, I just copy-paste it from somewhere.